courses/free-courses-en.md

```diff
@@ -250,6 +250,7 @@
 
 ### Compilers
 
+* [Compilers](https://www.youtube.com/playlist?list=PL6KMWPQP_DM97Hh0PYNgJord-sANFTI3i) - Ghassan Shobaki Computer Science Lectures (YouTube)
```

Comment:
"Computer Science Lectures "is not an author

Reply:
i've changed the name of the author